server/src/main/java/org/opensearch/identity/IdentityService.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Identity and access control for OpenSearch
  *
- * @opensearch.experimental
+ * @opensearch.api
```

Comment:
Please add `@InternalApi`, I don't think we expose this service anywhere, do we?

Reply:
Added both `@opensearch.internal` and `@InternalApi`. Core is the only place where this class is utilized. I think this should be marked as internal because no plugins should have exposure to the constructor or `initializeIdentityAwarePlugins`. If plugins need access to methods to get the current subject then I think we should explicitly create another class with API exposure which only exposes methods meant for plugins.